### PR TITLE
Counter

### PR DESCRIPTION
**Pull request by EmilyCook321**

### CODE REVIEW

File: client/src/components/Counter.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { ReactComponent as HeartIcon } from "../assets/icons/heart-icon.svg";
+import styled from "@emotion/styled";
+
+export default function Counter(props) {
```
Comment (JonasImm) [inline client/src/components/Counter.js:6]: You should think of naming your props for a better code understanding
```suggestion
export default function Counter(some name) {
```

If you like, you could also destructre you props here
```suggestion
export default function Counter({number}) {
```

File: client/src/components/Counter.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      <Heart>
+        <HeartIcon />
+      </Heart>
+      <output>{props.number}</output>
+    </CounterStyle>
+  );
+}
+
+Counter.propTypes = {
+  number: PropTypes.func,
```
Comment (JonasImm) [inline client/src/components/Counter.js:18]: number doesn't sound like a func, more like a number. If so, you should change the prop validation to 
```suggestion
  number: PropTypes.num,
```
If not, maybe you could rename number to something like `onSetNumber`

File: client/src/pages/NumberQuestionPage.js [hunk 3]
```diff
@@ -24,10 +25,11 @@ function NumberQuestionPage() {
...
 
   return (
-    <Numbermain>
-      <div>
-        <h1>Question {id}</h1>
-
+    <div>
+      <h1>Question {id}</h1>
+      <questionstyling>
+        <Counter number={incPoints} />
```
Comment (JonasImm) [inline client/src/pages/NumberQuestionPage.js:31]: Here you see, that number should be a `num`, not a `func` in your Counter.js

File: client/src/components/Counter.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+import { ReactComponent as HeartIcon } from "../assets/icons/heart-icon.svg";
+import styled from "@emotion/styled";
+
+export default function Counter(props) {
+  return (
+    <CounterStyle>
+      <Heart>
+        <HeartIcon />
+      </Heart>
+      <output>{props.number}</output>
```
Comment (JonasImm) [inline client/src/components/Counter.js:12]: If you would destructre, you could write your code like this:
```suggestion
      <output>{number}</output>
```

**Review by JonasImm — COMMENTED** [review]
Hey Emily, GJ 💯 👍 ,
I found some small things which I marked.

Also, if you work with your Kanban board, you should think of assigning yourself and the project to this PR, to keep better track of your progress:
image

File: client/src/components/Counter.js (hunk 2 above)
Comment (EmilyCook321) [reply client/src/components/Counter.js:19]: Thanks, good shout on this!

File: client/src/pages/NumberQuestionPage.js (hunk 3 above)
Comment (EmilyCook321) [reply client/src/pages/NumberQuestionPage.js:39]: Have implemented accordingly!

File: client/src/components/Counter.js (hunk 4 above)
Comment (EmilyCook321) [reply client/src/components/Counter.js:13]: Cheers, just implemented!

**Pull request merged**